liquibase-core/src/main/java/liquibase/command/core/CalculateChecksumCommandStep.java

```diff
@@ -20,9 +20,9 @@
 
     static {
         CommandBuilder builder = new CommandBuilder(COMMAND_NAME);
-        CHANGELOG_FILE_ARG = builder.argument("changelogFile", String.class).required()
+        CHANGELOG_FILE_ARG = builder.argument(CommonArgumentNames.CHANGELOG_FILE, String.class).required()
                 .description("The root changelog file").build();
-        URL_ARG = builder.argument("url", String.class).required()
+        URL_ARG = builder.argument(CommonArgumentNames.URL, String.class).required()
             .description("The JDBC database connection URL").build();
         DEFAULT_SCHEMA_NAME_ARG = builder.argument("defaultSchemaName", String.class)
```

Comment:
While they aren't as commonly used, defaultSchemaName and driverPropertiesFile are part of the set of common arguments across commands.

Reply:
I was contemplating changing around the other arguments as well. I'll do that in another PR to keep things nice and separate.